web/src/components/dropdown-poll-interval.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { SimpleSelect, SimpleSelectOption } from './SimpleSelect';
+import { useDispatch } from 'react-redux';
+import { queryBrowserSetPollInterval } from '../actions/observe';
+import { parsePrometheusDuration } from './console/utils/datetime';
+import { useTranslation } from 'react-i18next';
+
+export const DropDownPollInterval: React.FunctionComponent = () => {
+  const OFF_KEY = 'OFF_KEY';
+  const { t } = useTranslation('plugin__monitoring-plugin');
```

Comment:
We need to add a task to remember to update this to use the env var when this is included with ACM. cc @PeterYurkovich

Reply:
@PeterYurkovich Do you want me to wait for you [ACM PR](https://github.com/openshift/monitoring-plugin/pull/202) to merge before unholding this PR? Would that be easier? 

e.g. I can update 
`const { t } = useTranslation('plugin__monitoring-plugin');` to `const { t } = useTranslation(process.env.I18N_NAMESPACE);` and wait for [PR#202](https://github.com/openshift/monitoring-plugin/pull/202) to merge before unholding this PR. WDYT?

Reply:
This PR is safer than mine, so go ahead and merge this and I'll rebase and update the string